microceph/ceph/run.go

```diff
@@ -3,3 +3,23 @@ package ceph
 func cephRun(args ...string) (string, error) {
 	return processExec.RunCommand("ceph", args...)
 }
+
+func osdEnablePool(pool, app string) (string, error) {
+	return cephRun("osd", "pool", "application", "enable", pool, app)
+}
+
+func radosRun(args ...string) (string, error) {
+	return processExec.RunCommand("rados", args...)
+}
+
+func radosCreatePool(pool string) (string, error) {
```

Comment:
Please add a comment string for method (// radosCreatePool used to create dot pools i.e. pools that start with the "." prefix.)